docker-compose-services/varnish/README.md

```diff
@@ -0,0 +1,23 @@
+# Varnish
+
+This recipe allows you to configure a Varnish reverse proxy for your project.
+
+1. Copy [docker-compose.varnish.yaml](docker-compose.varnish.yaml) into your project's .ddev directory.
+2. Create a directory named _varnish_ in your project's .ddev directory.
+3. Put a file named _default.vcl_ in this directoy.
+4. `ddev start`
+
+For a quick start you can use the put the following contents into _default.vcl_:
```

Comment:
How about providing this as a file, so people don't miss it? That's what happened at least once in the issue.

Reply:
I've moved the example code to a real file so you can simply copy it. Additionally I've left a comment within the _default.vcl_ with link to a more advanced example.